review_routes/v3/results_routes.py

```diff
@@ -809,3 +933,152 @@ async def get_alignment_scores(
         result_list.append(result_obj)
     total_count = len(result_list)
     return {"results": result_data, "total_count": total_count}
+
+
+def agg_dicts(df):
+    # Extract relevant rows and create dictionaries for non-NaN 'assessment_id' and 'target'
+    filtered = df.dropna(subset=["baseline_id", "target"])
+    return [
+        {"revision_id": int(row["baseline_id"]), "target": row["target"]}
+        if row["baseline_score"] > 0.2
```

Comment:
i think the 0.2 can be a global variable in the file to make it easy to change in the future

Reply:
Yep. And now we're generating these results on the fly, we can also include this as a route parameter, in case the user wants to make their results more or less sensitive.